pkg/cmd/server/kubernetes/node.go

```diff
@@ -421,14 +423,29 @@ func (c *NodeConfig) RunProxy() {
 	default:
 		glog.Fatalf("Unknown proxy mode %q", c.ProxyConfig.Mode)
 	}
-	iptInterface.AddReloadFunc(proxier.Sync)
 
 	// Create configs (i.e. Watches for Services and Endpoints)
 	// Note: RegisterHandler() calls need to happen before creation of Sources because sources
 	// only notify on changes, and the initial update (on process start) may be lost if no handlers
 	// are registered yet.
 	serviceConfig := pconfig.NewServiceConfig()
-	serviceConfig.RegisterHandler(proxier)
+
+	unidlingLoadBalancer := ouserspace.NewLoadBalancerRR()
```

Comment:
Can I disable the unidler if I want to?  I think we need a way to be able to turn this off if necessary, which means a config flag or ProxyConfig.Mode or something similar.